README.md

```diff
@@ -59,6 +59,13 @@ This directive is always required.
 Maximum HTTP body size this module can generate SXG from. Default value is
 `67108864` (64 MiB).
 
+
+#### sxg\_cert\_path
+
+An absolute path in which nginx will generate and serve the CBOR-encoded certificate file.
+But make sure that the OCSP responder of the issuer of the certificate is accessible from your nginx server to get OCSP responses.
```

Comment:
s/the OCSP responder of the issuer of the certificate/the OCSP responder for the certificate/
The URI of the OCSP responder is written in the leaf certificate and it doesn't have to be the issuer CA.

Reply:
Done.